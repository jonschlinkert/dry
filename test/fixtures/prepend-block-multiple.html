{% extends "multiple" %}
{% block prepend="header" %}
header: "prepend-block-multiple.html"
{% endblock %}
{% block prepend="body" %}
body: "prepend-block-multiple.html"
{% endblock %}
{% block prepend="footer" %}
footer: "prepend-block-multiple.html"
{% endblock %}