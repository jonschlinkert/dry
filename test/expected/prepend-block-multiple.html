
head: "blocks/multiple.html"

header: "prepend-block-multiple.html"
header: "blocks/multiple.html"

body: "prepend-block-multiple.html"
body: "blocks/multiple.html"

footer: "prepend-block-multiple.html"
footer: "blocks/multiple.html"

foot: "blocks/multiple.html"
